lib/routes/devices-sessions.js

```diff
@@ -75,7 +75,7 @@ module.exports = (log, db, config, customs, push, devices) => {
 
         // Some additional, slightly tricky validation to detect bad public keys.
         if (payload.pushPublicKey && ! push.isValidPublicKey(payload.pushPublicKey)) {
-          throw error.invalidRequestParameter('invalid pushPublicKey')
+          return reply(error.invalidRequestParameter('invalid pushPublicKey'))
```

Comment:
This pattern of throwing a boom-compatible Error in the top level of the handler function no longer seems to work.  The docs seem to recommend either directly replying with the error, return returning a promise that will reject with the error.  This seems like the smaller change.